build/docker/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # This Stage prepares the user in the container and copies the files
```

Comment:
you really wanted to change this file?

Reply:
yes, because otherwise it will not work due to #810